homeassistant/components/jellyfin/config_flow.py

```diff
@@ -146,15 +142,15 @@ async def async_step_reauth_confirm(
 
     @staticmethod
     @callback
-    def async_get_options_flow(config_entry: ConfigEntry) -> OptionsFlow:
+    def async_get_options_flow(config_entry: JellyfinConfigEntry) -> OptionsFlow:
         """Create the options flow."""
         return OptionsFlowHandler(config_entry)
 
 
 class OptionsFlowHandler(OptionsFlow):
     """Handle an option flow for jellyfin."""
 
-    def __init__(self, config_entry: ConfigEntry) -> None:
+    def __init__(self, config_entry: JellyfinConfigEntry) -> None:
         """Initialize options flow."""
         self.config_entry = config_entry
```

Comment:
If you feel extra fancy you can inherit `OptionsFlowWithConfigEntry` instead, as you can remove the constructor

Reply:
Yes, very nice. Thanks.